assets/charts/components/velero/crds/backups.yaml

```diff
@@ -27,18 +26,24 @@ spec:
         apiVersion:
           description: 'APIVersion defines the versioned schema of this representation
             of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            internal value, and may reject unrecognized values. More info:
+            https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
           type: string
         kind:
           description: 'Kind is a string value representing the REST resource this
             object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            submits requests to. Cannot be updated. In CamelCase. More info:
+            https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
           type: string
         metadata:
           type: object
         spec:
           description: BackupSpec defines the specification for a Velero backup.
           properties:
+            defaultVolumesToRestic:
+              description: DefaultVolumesToRestic specifies whether restic
+                should be used to take a backup of all pod volumes by default.
+              type: boolean
```

Comment:
Since there are changes to the CRDs the release notes should include a way to apply CRDs first and then `lokoctl component apply`.